source/includes/entityref/_deduction.md

```diff
@@ -0,0 +1,105 @@
+# Deduction
+
+Entity that represents the Payroll provider's Deduction information.
```

Comment:
Make Payroll lowercase here because not at start of sentence.
Lowercase deduction